src/js/display/displayError.mjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * This will display an customizable error message to a container
+ * @param {string} message The message which will be displayed
+ * @param {*} [container=document.querySelector("main")] The container which will contain the error message
+ * @example
+ * ```js
+ * displayError("While trying to fetch the profile.");
+ * ```
+ */
+export default function displayError(message, container = document.querySelector("main")) {
+    const heading = document.createElement("h1");
+    heading.textContent = "An error ocurred!";
+
+    const messageContainer = document.createElement("p");
+    messageContainer.setAttribute("class", "text-danger fst-italic");
+    messageContainer.textContent = message;
+
+    container.innerHTML = "";
```

Comment:
You don't need to change this, and I did it until very recently too. It turns out this can lead to a memory overflow and a page crash since clearing HTML this way `does not destroy event listeners`. It's surprising that there is no good way to do this in 2022, the best I have found is this:

```js
function clearHTML() {
  while(this.firstChild){
    this.removeChild(this.firstChild);
  }
}

HTMLElement.prototype.clearHTML = clearHTML

// document.querySelector('header').clearHTML()
```